packages/perseus/src/widgets/__testdata__/expression.testdata.ts

```diff
@@ -71,7 +71,7 @@ export const expressionItem3Options: PerseusExpressionWidgetOptions = {
             value: "x+1",
         },
         {
-            considered: "incorrect",
+            considered: "wrong",
```

Comment:
I checked the published data. We definitely only use the set of `"correct" | "wrong" | "invalid"` as seen here: https://github.com/Khan/perseus/pull/582/files#diff-5737e5c75b244a64356894bb42c711539f1589cfbd9dcbbd4b22a3db8773d524R338